src/types/player.tsx

```diff
@@ -0,0 +1,13 @@
+import { AvatarId } from "types/avatars";
```

Comment:
Since there aren't an obnoxious amount of types for this app, instead of `src/types/<file>.tsx` for each category, maybe just a `src/types.tsx` file for ALL types?